backend/app.js

```diff
@@ -48,23 +72,23 @@ app.post('/signup', async (req, res) => {
     res.send('New user registered!')
 });
 
-app.post('/changeUsername', (req, res) => {
-    const newUsername = req.body.newUsername
-    res.send('Changed your username to: ' + newUsername);
-})
-
 // TODO T32: log in 
 app.post('/login', (req, res) => {
     res.status(200).send("Login endpoint under construction.");
 });
 
+app.post('/changeUsername', checkAuth, (req, res) => {
+    const newUsername = req.body.newUsername
+    res.send('Changed your username to: ' + newUsername);
+})
+
```

Comment:
I believe it's just a placeholder. I think it's showing up as added because I originally had the method for middleware as app.use() and swapped /changeUsername and /login since the I read that the order matters. I can change it back if needed.

Reply:
Ok i see, you don't need to change it back